src/config/index.ts

```diff
@@ -40,17 +32,6 @@ export default {
           process.env.SQS_PUBLISHER_DATA_QUEUE_URL ||
           'http://localhost:4566/000000000000/pocket-publisher-data-queue',
       },
-      listDeleteQueue: {
```

Comment:
@jpetto is this not used?

Reply:
Gotcha, just makes me wonder what it was... as well, but im cool with rolling forward!

Reply:
@bassrock i can't even find this queue in AWS! (nor can i see it created in the `.aws` directory in `main`.) looks like it was [created by kelvin](https://github.com/Pocket/list-api/commit/091bd5062e25d68401d6e396bd3b22ee460426f8) back when this repo had a different structure...

Reply:
since i am not proficient in this repo, i would love confirmation here 😅 

according to the [existing config](https://github.com/Pocket/list-api/blob/main/.aws/src/config/index.ts#L43), i believe the queue name in infra should be `ListAPI-Prod-Sqs-Batch-Delete-Consumer-Queue`. i don't see any infra code creating this queue, however. and i don't see a queue in our prod AWS account matching the name above.

on top of that, on the `main` branch, i can't find any reference outside of the config file referenced here to the variable name `listDeleteQueue`.

Reply:
@jpetto thats safe to delete then. My guess is it was left over from intial account deletion work and the team went a different direction